app/static/js/admin/users/users.js

```diff
@@ -11,6 +11,25 @@ $(function() {
         }
     });
 
+    /* =======================================================================
+     * Table Highlight and Click
+     * ==================================================================== */
+    $('td').hover(function() {
+        var t = parseInt($(this).index()) + 1;
```

Comment:
can we use a better varaible name than `t`?  what does this represent?  (maybe `columnNo`?)